### PR TITLE
Add docs for installing on GCP

### PR DESCRIPTION
**Pull request by aclevername**
Part of #72 

The diff is a mess because of shuffling of partials. best to clone locally and browser the docs

### CODE REVIEW

File: docs/main/04-guides/01-installing-kratix/04-others.mdx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 ---
 description: Run Kratix
-slug: /main/guides/installing-kratix
-title: Multi cluster
+slug: /main/guides/installing-kratix-others
+title: Others (KinD etc)
```
Comment (SaphMB) [inline docs/main/04-guides/01-installing-kratix/04-others.mdx:4]: Maybe "Other Tools"?

Comment (aclevername) [reply docs/main/04-guides/01-installing-kratix/04-others.mdx:4]: yeah, maybe `Other Providers`? Thinking for something that covers openshift/civo/kind/minikube

**Comment by kirederik** [comment]
I imagined (maybe wrong) this doc page with a "copy-paste and this will work" flow;

wdyt about adding the commands to:
- create a platform cluster using the gcloud cli
- create a gcs bucket with s3 compatibility w/ the gcloud cli
- create a worker cluster using the gcloud cli
- copy and paste of destination, statestore and flux bits

**Comment by aclevername** [comment]
> I imagined (maybe wrong) this doc page with a "copy-paste and this will work" flow;
> 
> wdyt about adding the commands to:
> 
> * create a platform cluster using the gcloud cli
> * create a gcs bucket with s3 compatibility w/ the gcloud cli
> * create a worker cluster using the gcloud cli
> * copy and paste of destination, statestore and flux bits

Lets chat about this on Monday. I think maintaining docs on creating the clusters seems unneccessary when we can just link to the actual IaaS's docs. I'm open to the 2nd (is it more compelling is my only thought?). 4th is only possible if we got with the 2nd I guess

**Pull request merged**